src/scripts/components/story-media/story-media.styl

```diff
@@ -51,15 +89,19 @@
 
 .slider
   display: flex
-  height: emCalc(550px)
+  height: emCalc(600px)
```

Comment:
600px doesn't sound responsive. can we use something relative to screen size here?